docker/local/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM clojure:lein-alpine as build
+
+RUN apk --no-cache add \
+    git \
+    nodejs \
+    nodejs-npm
+
+RUN npm install -g bower stylus
+
+WORKDIR /netrunner
+COPY . .
+RUN lein deps
+RUN bower --allow-root install
+RUN stylus src/css -o resources/public/css/
+RUN lein cljsbuild once prod
+RUN lein uberjar
+
+# Final image
+FROM clojure:lein-alpine
```

Comment:
Does this work? I've never done it this way but it seems much better if indeed it cuts dependency checking time down (once the "build" stuff has run once and doesn't change)

Reply:
Not clear if this speeds up the dependency checking. I'm doing it to get rid of all the build artifacts; it reduces the final image size by ~200MB.

Docker does seem to cache the intermediate layers, so I don't think it's rebuilding unnecessarily.

Reply:
Okay cool that's a win then. I can grab this at some point and see if splitting up the dockerfiles helps some.